src/OpenLoco/src/GameCommands/GameCommands.h

```diff
@@ -990,6 +1340,36 @@ namespace OpenLoco::GameCommands
         return doCommand(GameCommand::vehicleOrderInsert, regs);
     }
 
+    struct VehicleOrderInsertArgs
+    {
+        static constexpr auto command = GameCommand::vehicleOrderInsert;
+
+        VehicleOrderInsertArgs() = default;
+
+        explicit VehicleOrderInsertArgs(const registers& regs)
+            : head(EntityId(regs.di))
+            , rawOrder(static_cast<uint64_t>(regs.cx) << 32 | regs.eax)
```

Comment:
put more brackets around this `(static_cast<uint64_t>(regs.cx) << 32) | regs.eax`